src/relay/ir/expr.cc

```diff
@@ -72,8 +72,8 @@ Constant::Constant(runtime::NDArray data, Span span) {
 
 TVM_REGISTER_NODE_TYPE(ConstantNode);
 
-TVM_REGISTER_GLOBAL("relay.ir.Constant").set_body_typed([](runtime::NDArray data) {
-  return Constant(data);
+TVM_REGISTER_GLOBAL("relay.ir.Constant").set_body_typed([](runtime::NDArray data, Span span) {
```

Comment:
Thank you for point out this concern. :D
Just a gentle reminder that we do have the default value of span for these types. Because currently the constructors in header ([include/tvm/relay/expr.h](https://github.com/apache/tvm/blob/b2058f4dd2e0ae1fc5ab51ac9f84b372a389a65a/include/tvm/relay/expr.h#L107)) have already been assigned with Span() to its attribute. We simply make span attribute be modifiable here.

Reply:
okay cool--just wondering, since most of the passes are in C++, is the plan to invoke this via PackedFunc between each pass?

Reply:
Hi areusch,

Thanks for replying!
About the question "Is the plan to invoke this via PackedFunc between each pass?". 

No, this SpanFiller will not be triggered between each pass. Perhaps there are some misunderstandings. Please allow me to describe more. :D 

This PR, and the SpanFiller is for "frontend conversion" specifically. Take the upcomming PR of TFLite span filling for example, the function "set_span()" is invoked after a [conversion of an operator only](https://github.com/chunit-quic/tvm/blob/cee1703b4be6ac0e1ed3fb7c493432c95e7a5231/python/tvm/relay/frontend/tflite.py#L282).

About the [span filling mechanism for the passes](https://discuss.tvm.apache.org/t/pre-rfc-tvm-explorer-infrastructure/13457#pass-source-information-builder-6), we will use the SequentialSpan and source information builder to deal with. They are another set of classes and function to propagate span between passes specifically.

If there is any confusing part please feel free to let me know, I will try to make it clear. :)
Thank you!

Reply:
ah okay that makes sense now, i think the point of this PR is very clear to me. thanks for explaining and sorry for the long delay here!